WooCommerce/Classes/Yosemite/DefaultStoresManager.swift

```diff
@@ -506,7 +507,9 @@ private extension DefaultStoresManager {
     ///
     func updateAndReloadWidgetInformation(with siteID: Int64?) {
         // Token to fire network requests
-        keychain.currentAuthToken = sessionManager.defaultCredentials?.authToken
+        if case let .wpcom(username: _, authToken: authToken, siteAddress: _) = sessionManager.defaultCredentials {
```

Comment:
Nit:
```suggestion
        if case let .wpcom(_, authToken, _) = sessionManager.defaultCredentials {
```

Reply:
Done as part of b231bcef7029ee63fab0c1e723c9e645ed2e16e1